op-chain-ops/deployer/opcm/contract.go

```diff
@@ -0,0 +1,83 @@
+package opcm
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+type Contract struct {
+	addr   common.Address
+	client *ethclient.Client
+}
```

Comment:
This is fine for now, but lets' separate remote-client and local deployer code, once we have forking functionality.